### PR TITLE
Update Sentinel dVPN

### PR DESCRIPTION
**Pull request by Dimokus88**

### CODE REVIEW

File: Sentinel-dVPN-node/deploy.yml [hunk 1]
```diff
@@ -1,37 +1,39 @@
 ---
 version: "2.0"
 endpoints:
-  your_endpoint_name:  # # WARRING! SPECIFY UNIQ ENDPOINT NAME! ONLY LOWER CASE SYMBOLS!
+ uniq_name_endpoint: # unique name, for example "dvpn_dimokus:", and enter this name in line 23 and 28.
```
Comment (arno01) [inline Sentinel-dVPN-node/deploy.yml:4]: add one space before `uniq_name_endpoint: ` line :-)

File: Sentinel-dVPN-node/main.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    ["HOURLY_PRICES"]="hourly_prices"
+)
+
+# Update each variable in config.toml.
+for var in "${!config_mappings[@]}"; do
+    update_config "$var" "${config_mappings[$var]}" "$CONFIG_PATH"
+done
+
+REMOTE_URL="$IPV4_ADDRESS:$REMOTE_PORT"
+sed -i.bak -e "s|^remote_url *=.*|remote_url = \"https://$REMOTE_URL\"|;" "$CONFIG_PATH"
```
Comment (arno01) [inline Sentinel-dVPN-node/main.sh:59]: why don't you leverage the `config_mappings` and `update_config` function? :)

**Review by arno01 — CHANGES_REQUESTED** [review]
Fantastic work!

Couple of more additions pls

File: Sentinel-dVPN-node/deploy.yml (hunk 1 above)
Comment (Dimokus88) [reply Sentinel-dVPN-node/deploy.yml]: Fix it

**Review by arno01 — APPROVED** [review]
Dimokus tested.

**Review by andy108369 — APPROVED** [review]

**Pull request merged**